examples/certs/docker/docker.ext

```diff
@@ -0,0 +1,10 @@
+authorityKeyIdentifier=keyid,issuer
+basicConstraints=CA:FALSE
+keyUsage = digitalSignature, nonRepudiation, keyEncipherment, dataEncipherment
+subjectAltName = @alt_names
+
+[alt_names]
+DNS.1 = project-oak.local
```

Comment:
You can probably remove everything except for the static cloud IP address.

Reply:
In this case we will not be able to test Docker images locally.